src/newSeed/baseStage.ts

```diff
@@ -21,6 +21,7 @@ export interface IWizardState {
   requiredSeedDeposit?: number;
   requiredSeedFee?: number;
   primeDaoAddress?: Address;
+  seedAdminAddress?: Address;
```

Comment:
why add this? why not just use `seedConfig.seedDetails.adminAddress`?

Reply:
Stage8 also displays the seed admin address, but the seedConfig state gets cleared in stage7 upon creating a seed. So, I added that information to the wizard state to allow it to persist until stage8.